snuba/datasets/entities/sessions.py

```diff
@@ -304,3 +316,14 @@ def get_query_processors(self) -> Sequence[QueryProcessor]:
                 {"bucketed_started": "started"}, ("started", "received")
             ),
         ]
+
+
+def check_if_crash_rate_alert_subscription(query: Query) -> bool:
+    """
+    Checks if the provided query is for a Crash Rate Alert subscription based on the if one of
+    the aggregations matches the CRASH_RATE_ALERT_PATTERN
+    """
+    for col in query.get_selected_columns():
+        if col.name and bool(CRASH_RATE_ALERT_PATTERN.match(col.name)):
```

Comment:
Relying on a specific alias to make this work seems extremely brittle.

Reply:
I agree.. I will think of a better solution